code/src/index.css

```diff
@@ -1,13 +1,50 @@
+*, *::before, *::after {
+  box-sizing: border-box;
+}
+
+html {
+  height: 100%;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Inconsolata', monospace;
+  background-color: black;
+  color: #4CFF42;
+  padding: 35px;
+  height: 100%;
+}
+
+.swal-modal {
+  background-color: #000;
+  border: 2px solid #4CFF42;
+}
+
+.swal-button {
+  padding: 7px 19px;
+  border-radius: 2px;
+  color: #000;
+  background-color: #4CFF42;
+  font-size: 12px;
+  border: 1px solid #4CFF42;
+}
+
+.swal-button:hover {
+  background-color: black;
+}
+
+.swal-content__input[type="text"], .swal-content__input[type="text"]:focus {
+  font-family: 'Inconsolata', monospace;
+  background-color: #000;
+  color: #4CFF42;
+  outline: #4CFF42;
+  border: 1px solid #4CFF42;
```

Comment:
Why are you styling swal with regular CSS? Can't you use styled components for this?